app/metadata.js

```diff
@@ -1,35 +1,24 @@
 const statsdClient = require('./statsd');
 const domino = require('domino');
-const urlparse = require('url');
-const {getMetadata} = require('page-metadata-parser');
+const {getMetadata, makeUrlAbsolute} = require('page-metadata-parser');
 require('isomorphic-fetch');
 
-function makeUrlAbsolute(base, relative) {
-  const relativeParsed = urlparse.parse(relative);
-
-  if (relativeParsed.host === null) {
-    return urlparse.resolve(base, relative);
-  }
-
-  return relative;
-}
-
 function getDocumentMetadata(url, window) {
   const doc = window.document;
-  const metadata = getMetadata(doc);
+  const metadata = getMetadata(doc, url);
 
   const responseData = {
-    url,
-    original_url: url,
+    url: metadata.url,
+    original_url: metadata.url,
```

Comment:
This feels wrong. Mainly, we're returning the same `metadata.url` value twice, which seems confusing, but also, the value returned from `metadata.url` may not be the original URL (if it was canonical-ized or something). Shouldn't this just be the `url` that the user specified echoed back to them?
```js
url: metadata.url,
original_url: url,
...
```